### PR TITLE
Add image label to container metrics

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

**Comment by mwielgus** [comment]
cc: @piosz

File: sinks/api/v1/supported_labels.go [hunk 1]
```diff
@@ -54,12 +54,17 @@ var (
 		Key:         "host_id",
 		Description: "Identifier specific to a host. Set by cloud provider or user",
 	}
+	LabelImage = LabelDescriptor{
+		Key:         "image",
+		Description: "User-defined image name that is run inside the container",
```
Comment (vishh) [inline sinks/api/v1/supported_labels.go]: How about "Container base image" instead?

File: sinks/api/decoder.go [hunk 2]
```diff
@@ -98,6 +98,8 @@ func (self *defaultDecoder) getContainerMetrics(container *source_api.Container,
 		return nil
 	}
 	labels[sinksV1Api.LabelContainerName.Key] = container.Name
+	labels[sinksV1Api.LabelImage.Key] = container.Image
```
Comment (vishh) [inline sinks/api/decoder.go]: Can you add this in sinks/api/v1/decoder.go as well? The code is in a state of flux. Thats the reason for duplication.

**Comment by vishh** [comment]
Just a few comments. Otherwise LGTM

**Comment by vishh** [comment]
ok to test

File: sinks/api/v1/supported_labels.go (hunk 1 above)
Comment (mwielgus) [inline sinks/api/v1/supported_labels.go]: Done

File: sinks/api/decoder.go (hunk 2 above)
Comment (mwielgus) [inline sinks/api/decoder.go]: Added here and to the cache.

**Comment by vishh** [comment]
LGTM. Thanks for the PR @mwielgus

**Pull request merged**